recipes/radio_beam/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "radio_beam" %}
+{% set version = "0.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3b3b3cfcc17d61df469c9e45d7f7fc993b902f3f2c076279071fd984a6e5763d" %}
+
+package:
+  name: radio-beam
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
```

Comment:
Deliberately not using pip as need to use the ``--offline`` option due to custom setuptools code in this package.

Reply:
This also means for now we can't use python: noarch

Reply:
I'm not one of the "official" reviewers but, out of curiosity, what do you mean by "custom setuptools code"?

Reply:
Possibly your intial commit did't work because you didn't include `setuptools` in the build requirements.

Reply:
This (and other astropy packages) require an --offline flag to be passed to avoid a remote download but this needs to be passed for egg_info too, which pip doesn't allow. However I have a commit I'm about to push that will add a patch instead so I can use pip.